test/common_device_type.py

```diff
@@ -229,6 +278,60 @@ def only_fn(slf, device, *args, **kwargs):
         return only_fn
 
 
+# Lists of Torch types
+DOCUMENTED_TENSOR_TYPES = [torch.float64, torch.float32, torch.float16,
```

Comment:
The source of truth of these lists should only exist in one place in python testing land, e.g. here or https://github.com/pytorch/pytorch/blob/98ccae09afc8e1d67ff62b3e6fd0ecec21328993/torch/testing/__init__.py#L89-L102 (i.e. one just uses the other if we need both).

Reply:
Sounds good.